src/test/init.js

```diff
@@ -26,7 +62,7 @@ test('init()', nest => {
     const mutators = ['a', 'b', 'c'].map(
       (i) => (o, { instance }) => { instance[i] = i; });
 
-    const actual = init(...mutators)();
+    const actual = init(mutators[0], mutators.slice(1, 3))();
```

Comment:
100% agree. Thank you for binging the simplification topic to the surface. We somehow missed that point.
I created the new issue for discussion here: https://github.com/stampit-org/stampit/issues/221